scenarios/kubernetes_e2e.py

```diff
@@ -442,6 +459,9 @@ def create_parser():
         '--test', default='true', help='If we need to set --test in e2e.go')
     parser.add_argument(
         '--up', default='true', help='If we need to set --up in e2e.go')
+    parser.add_argument(
+        '--multiple-federations', default=False, action='store_true',
```

Comment:
Please just pass this flag into kubetest, like we do build.

This script is not intended to perform a bunch of complicated logic.